miranda/convert/_utils.py

```diff
@@ -2,50 +2,49 @@
 import json
 import logging.config
 import os
-import shutil
 from pathlib import Path
-from typing import Dict, List, Optional, Sequence, Union
+from typing import Dict, Optional, Union
 
-import dask.config
 import netCDF4
 import numpy as np
 import regionmask
-import xarray
 import xarray as xr
 import zarr
 from clisops.core import subset
-from dask import compute
-from dask.diagnostics import ProgressBar
-from xarray import Dataset
 from xclim.core import calendar, units
 from xclim.indices import tas
 
-from miranda.gis.subset import subsetting_domains
 from miranda.scripting import LOGGING_CONFIG
-from miranda.utils import chunk_iterables
-
-from ._data import project_institutes, xarray_frequencies_to_cmip6
 
 logging.config.dictConfig(LOGGING_CONFIG)
 
-
-dask.config.set(local_directory=f"{Path(__file__).parent}/dask_workers/")
+__all__ = [
+    "get_chunks_on_disk",
+    "add_ar6_regions",
+    "daily_aggregation",
+    "delayed_write",
+    "variable_conversion",
+]
 
 LATLON_COORDINATE_PRECISION = dict()
 LATLON_COORDINATE_PRECISION["era5-land"] = 4
 
 VERSION = datetime.datetime.now().strftime("%Y.%m.%d")
 
 
-# Needed pre-processing function
-def _drop_those_time_bnds(dataset: xr.Dataset) -> xr.Dataset:
-    if "time_bnds" in dataset.variables:
-        return dataset.drop_vars(["time_bnds"])
-    return dataset
+def get_chunks_on_disk(nc_file: Union[os.PathLike, str]) -> dict:
+    """
 
+    Parameters
+    ----------
+    nc_file: Path or str
 
-def get_chunks_on_disk(ncfile: Union[os.PathLike, str]) -> dict:
-    ds = netCDF4.Dataset(ncfile)
```

Comment:
I think eventually we could eventually try to add an option that would multiply disk chunks by a factor to reach a good 'dask' chunk size (typically bigger than on disk I think), but I think I would leave it as is until we hit a performance issue down the road sometime